src/pages/about-us/history.tsx

```diff
@@ -1,15 +1,27 @@
 import * as React from "react"
 import { PageProps, graphql } from "gatsby"
-import { Stack, Typography } from "@mui/material"
+import { Stack, Typography, useTheme, Grid } from "@mui/material"
```

Comment:
I normally use the Grid v2 from MUI because it has added functionality but this is fine for now! Could you update it in a later PR?
```
import Grid from "@mui/material/Unstable_Grid2/Grid2"
```